test/mailers/notifier_test.rb

```diff
@@ -1,10 +1,18 @@
 require 'test_helper'
 
 class NotifierTest < ActionMailer::TestCase
+  fixtures :lists
 
   def setup
-    recipients = User.all.pluck :email
-    options = {recipients: recipients, completed: 0, remaining: 5}
+    list = lists(:first)
+
+    options = {
+      recipients: ['vasyll@tada.com'],
+      completed_todos: {list => list.items},
```

Comment:
@sergeyradkevich, I've tried to do it in a such way, but it doesn't work (template error). I think it's because `list` is interpreted like a symbol, not like an object and then any list's fields like `title` are undefined.